ta/pkcs11/src/attributes.h

```diff
@@ -179,6 +179,15 @@ static inline enum pkcs11_rc get_u32_attribute(struct obj_attrs *head,
 	return rc;
 }
 
+/*
+ * Return true all attributes from the reference are found and match value
```

Comment:
"Return true **if** all attributes from the reference ..."

Reply:
Changed